doc/nrf/libraries/networking/aws_fota.rst

```diff
@@ -49,6 +49,12 @@ Creating a FOTA job
       }
 
    Replace the *bucket_name* with the name of your bucket.
+
+.. note::
+   The policy example is only intended for development environments.
+   AWS Best Practises recommends that the **principal** is specified in the Bucket Policy to prevent ananymous users access to S3.
```

Comment:
```suggestion
   AWS Best Practices recommends that the **principal** is specified in the Bucket Policy to prevent anonymous users access to S3.
```